code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -84,6 +84,7 @@
 	playsound(victim,'sound/weapons/alien_claw_block.ogg', 75, 1)
 
 	succeed_activate()

```

Comment:
Not a problem for this PR but a note. Couldn't succeed_activate just set the cooldown instead?

Reply:
Sounds sensible. If you consume plasma in general you want the cooldown activated if there's any.
Should poke @spookydonut about it, though, since this system is his design.